indico/modules/events/surveys/controllers/management.py

```diff
@@ -112,10 +114,17 @@ def _process(self):
 
 
 class RHManageSurveyQuestionnaire(RHManageSurvey):
+    def _create_preview_form(self):
+        return create_preview_form(self.survey.questions)()
+
     def _process(self):
         field_types = get_field_types()
+        preview_form = self._create_preview_form()
+
         return WPManageSurvey.render_template('manage_questionnaire.html', self.event,
-                                              survey=self.survey, field_types=field_types)
+                                              survey=self.survey, field_types=field_types,
+                                              preview_form=preview_form,
+                                              preview_widget_attrs={'disabled': True})
```

Comment:
Why define this here? You could simply pass the dict when calling `form_row` in the template